packages/proto/definitions/file_config.proto

```diff
@@ -3,20 +3,39 @@ syntax = "proto3";
 package juno.file_service.config;
 
 import "identifiers.proto";
+import "project.proto";
+import "file_bucket.proto";
+import "file.proto";
 
-service FileServiceFileServiceConfigDbService {
+service FileServiceConfigDbService {
   rpc getConfig(GetFileServiceConfigRequest) returns (FileServiceConfig);
   rpc createConfig(CreateFileServiceConfigRequest) returns (FileServiceConfig);
   rpc deleteConfig(DeleteFileServiceConfigRequest) returns (FileServiceConfig);
   rpc updateConfig(UpdateFileServiceConfigRequest) returns (FileServiceConfig);
 }
 
-message FileServiceConfig {}
+message FileServiceConfig {
+  int64 id = 1;
+  repeated file_bucket.Bucket buckets = 2;
```

Comment:
The build failure is coming from here, this (and all the others) should be `config.Bucket`